modules/150-user-authn/docs/FAQ.md

```diff
@@ -27,26 +27,20 @@ spec:
   - everyone
   - admins
   whitelistSourceRanges:
-  - 1.1.1.1
+  - 1.1.1.1/32
   - 192.168.0.0/24
 ```
 
-{% endraw %}
-
 After the `DexAuthenticator` custom resource is created in the cluster, Kubernetes will make the necessary deployment, service, ingress, secret in the specified namespace.
 Add the following annotations to your app's Ingress resource to connect your application to dex:
 
-{% raw %}
-
 ```yaml
 annotations:
   nginx.ingress.kubernetes.io/auth-signin: https://$host/dex-authenticator/sign_in
   nginx.ingress.kubernetes.io/auth-url: https://my-cool-app-dex-authenticator.my-cool-namespace.svc.{{ cluster domain, e.g., | cluster.local }}/dex-authenticator/auth
   nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-User,X-Auth-Request-Email,Authorization
 ```
```

Comment:
2. Connect the app to Dex.

   To do this, add the following annotations to the Ingress resource of the application: